### PR TITLE
Fix no-chacha and no-poly1305

### PR DESCRIPTION
**Pull request by mattcaswell**
##### Checklist

- [ ] documentation is added or updated
- [x] tests are added or updated

### CODE REVIEW

File: test/cipherlist_test.c [hunk 1]
```diff
@@ -72,7 +72,7 @@ static const uint32_t default_ciphers_in_order[] = {
     TLS1_CK_DHE_RSA_WITH_AES_256_GCM_SHA384,
 # endif
 
-# if !defined OPENSSL_NO_CHACHA && !defined OPENSSL_NO_POLY1305
+# if !defined(OPENSSL_NO_CHACHA) && !defined(OPENSSL_NO_POLY1305)
```
Comment (levitte) [inline test/cipherlist_test.c]: Just so I understand correctly, this is purely stylistic, right?

Comment (mattcaswell) [reply test/cipherlist_test.c]: Yes

**Comment by richsalz** [comment]
We should put a simple version of run-checker in our tools repo so that people can try things themselves.

**Comment by levitte** [comment]
Actually, the run-checker script is fairly simple, and can run out of the box.  It might need a couple of tweaks before being ready for public consumption.  I can look into it.

**Comment by mattcaswell** [comment]
Ooops. Forgot to close this. This has been merged.

**Pull request closed without merging**